src/languages/swift.js

```diff
@@ -96,7 +84,52 @@ export default function(hljs) {
         { begin: /\b0b([01]_*)+\b/ },
       ]
   };
-  SUBST.contains = [NUMBER];
+
+  // https://docs.swift.org/swift-book/ReferenceManual/LexicalStructure.html#grammar_string-literal
+  var ESCAPED_CHARACTER = (rawDelimiter = "") => ({
+    className: 'subst',
+    variants: [
+      { begin: concat(/\\/, rawDelimiter, /[0\\tnr"']/) },
+      { begin: concat(/\\/, rawDelimiter, /u\{[0-9a-fA-F]{1,8}\}/) }
+    ]
+  });
+  var ESCAPED_NEWLINE = (rawDelimiter = "") => ({
+    className: 'subst',
+    begin: concat(/\\/, rawDelimiter, /[\t ]*(?:[\r\n]|\r\n)/)
+  });
+  var INTERPOLATION = (rawDelimiter = "") => ({
+    className: 'subst',
+    begin: concat(/\\/, rawDelimiter, /\(/),
+    end: /\)/
```

Comment:
```suggestion
    className: 'subst',
    keywords: SWIFT_KEYWORDS,
    begin: concat(/\\/, rawDelimiter, /\(/),
    end: /\)/
```

Don't we need keywords here?

Reply:
That's something I was wondering too. The current grammar is a bit too general when it comes to keywords, and often misidentifies them. In this case, String interpolation only supports expressions, not statements, so most of the keywords in `SWIFT_KEYWORDS` shouldn't occur here.

Reply:
Yes, I was flagging the obvious change in behavior this will have.

>  String interpolation only supports expressions, not statements, so most of the keywords in SWIFT_KEYWORDS shouldn't occur here.

If that's the case then this removal sounds reasonable...  (although if it breaks any of the existing tests it'll need to be looked at more carefully).